front-end-project/src/components/Notes/Notes.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import './Notes.css';
+import Note from '../Note/Note';
+
+
+const Notes = props => {
+
+    return (
+        <div>
+            <h2>Your Notes:</h2>
+            {props.myNotes.map(note => {
```

Comment:
If this component tries to render before the get call is made, or if someone deletes all the notes in the API, it will break with an error because we can't map over an empty array. How can you give the component the ability to display a different message if there are no notes?